src/test/scala/com/raquo/airstream/core/TransactionSpec.scala

```diff
@@ -69,6 +69,7 @@ class TransactionSpec extends UnitSpec {
         new Transaction(_ => {
           throw new Exception("Random error in transaction")
         })
+        ()
```

Comment:
I would prefer to disable `-Ywarn-value-discard`. It's useless because it's [not applied consistently](https://github.com/scala/bug/issues/11379) (one of many examples: `new Transaction` in `ConcurrentEventStream.internalEventObserver`), and just pollutes the code with spurious `()`. While we could change the API of Transaction to return Unit, I like it the way it is. Also, I can't change the native JS APIs that we're using that don't return Unit.

Reply:
It was quite helpful in our big project at work. But will disable it